recipes/krbcontext/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "krbcontext" %}
+{% set version = "0.10" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/krbcontext-{{ version }}.tar.gz
+  sha256: f7f1bdc33a86f18f3d119a631eb250d1389378fdaea1a983eef8ef0996d64ab8
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python-gssapi
+    - python >=3.6
```

Requiring the `__unix` virtual package should get you what you want

```suggestion
    - python >=3.6
    - __unix
```

---

*amazing*

It would be embarrassing to say how long I spent looking for that option in the docs. Let's just say that it was a long time. Much appreciated!!!

---

Sometimes you have to know how to look because you've been there before. 😉